nlprule/src/bin/test.rs

```diff
@@ -19,8 +21,8 @@ fn main() {
     env_logger::init();
     let opts = Opts::parse();
 
-    let tokenizer = Tokenizer::new(opts.tokenizer).unwrap();
-    let rules_container = Rules::new(opts.rules).unwrap();
+    let tokenizer = Arc::new(Tokenizer::new(opts.tokenizer).unwrap());
```

Comment:
Iff the above API changes are implemented, the Arc should be able to be avoided. Imho `Spellcheck` should parse the file and maybe use a `Tokenizer` _reference_ only when extracting items.